src/js/ui/locale/locale.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Translate messages
+ */
+class Locale {
+    constructor(locale) {
+        this._locale = locale;
+    }
+
+    /**
+     * localize message
+     * @param {string} message - message who will be localized
+     * @returns {string}
+     */
+    localize(message) {
+        return (this._locale[message]) ? this._locale[message] : message;
```

Comment:
Is this parenthesis necessary? I think it's better using  short-circuit evaluation than conditional ternary operator.
```js
return this._locale[message] || message;
```

Reply:
@dongsik-yoo, It is a good idea. I refactored it.